server.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.static('./public'));
+app.listen(PORT, function(){
+  console.log('server 3000');
```

Comment:
This should be a template literal because you're not always going to use port 3000